azavea/district-builder-dtl-pa
# Personal statement copy change, add endorsers, update links

## Overview

Small changes requested by C70 described here:
* https://www.pivotaltracker.com/story/show/164431096
* https://www.pivotaltracker.com/story/show/164431088
* https://www.pivotaltracker.com/story/show/164803109

### Checklist


### Demo

![dtl_personal_statement](https://user-images.githubusercontent.com/2926237/54935846-86fb1000-4ef7-11e9-8b8c-3d4de8a144aa.png)

## Testing Instructions

* `./scripts/update` and `./scripts/server`
* Make sure links on landing page are correct
* Go to submit a valid plan and make sure updated copy and link are correct for Personal Statement
* Verify there is a new optional "Endorsers" field in the submission form and that the copy/formatting/link is correct for that (link is currently broken but @tgilcrest says that's fine for now since they can change that on their end)
* Submit plan with endorsers and verify that that text shows up appropriately in the database like so:
```
district_builder=# select endorsers from redistricting_plansubmission where end
orsers is not null;
    endorsers
------------------

 Jane: j@j.com   +
 Grigory: g@g.com
(2 rows)
```
